### DISCUSSION TITLE
How do I use Kubernetes ServiceAccounts to access my cloud resources?

### DISCUSSION THREAD

**Discussion by SandraGH5** [flyteorg/flyte#1079] [Q&A] [answered]

**Comment by SandraGH5** [comment, marked as answer]
Please visit "Accessing Cloud Resources" under Cluster Deployment at https://docs.flyte.org/projects/cookbook/en/latest/auto/deployment/cluster/access_cloud_resources.html#.

**Comment by davidmirror-ops** [comment]
For updated guidance (AWS-specific) see:  https://github.com/davidmirror-ops/flyte-the-hard-way/blob/main/docs/aws/03-roles-service-accounts.md
